capabilities/application-connectivity.md

```diff
@@ -0,0 +1,48 @@
+---
+displayName: Application Connectivity
+epicsLabels:
+  - area/application-connector
+---
+
+## Scope
+
+The Application Connectivity capability supports the extensibility and integration between applications in a cloud-native way using the best available solutions.
+It provides integration patterns and necessary tooling for establishing a trusted connection between Kyma and third-party applications.
+
+The goal is to enable the integration of various systems in a coherent way, where the management and usage of APIs and Events are standardized and allow to extend such systems in a natural, cloud-native way.
+The unification of the connected system enables fast development which unlocks brand new possibilities for extending and customizing the existing solution in a modern, cloud-based, way.
+
+A new way of system scalability is available. The event mechanism and simple access to the exposed API is a foundation for moving the workload from a legacy system to Kubernetes.
+
+An additional benefit is that you can mesh different systems using the language of your choice.
+
+The integration must be as simple as possible.
+
+## Vision
+
+* API Registry and Discoverability
+
+    * Connected application can register its APIs and Event catalogs. 
+    * The registration contains the configuration of the endpoints together with required security credentials, documentation of the API and Event catalog based on open standards like OpenAPI and AsyncAPI, and additional documentation.
+    * Registered APIs and Event catalogs will be exposed as virtual services in Service Catalog.
+    * The registration of the application contains the required metadata, like health endpoints, localization, version.
+     
+* Orchestration
```

Comment:
Orchestration is a little bit blurry. I would like to see: it is possible to write the cloud native, scallable extension that can consume (or be triggered by) any event coming from the connected application and can call any connected application API without additional configuration effort. The App registry information is available for developers in the tooling.

Reply:
I have delivered a different description. Still a little bit blurry. It requires further discussion.